src/main/java/com/hubspot/jinjava/el/ext/AstRangeBracket.java

```diff
@@ -41,15 +41,24 @@ public Object eval(Bindings bindings, ELContext context) {
       return evalString((String) base, bindings, context);
     }
 
-    Object start = property.eval(bindings, context);
+    Iterable<?> baseItr;
```

Comment:
you could use a ternary here

Reply:
I need to have a better eye out for refactoring code that I come across.